src/components/popups/InspectTask.tsx

```diff
@@ -124,6 +127,7 @@ const InspectTask = ({ isOpen, onClose, task }) => {
           },
         }
       );
+      setIsLoading(false);
```

Comment:
does not get reset on error: reset `setIsLoading(false)` after try-catch block

Reply:
Moved down